src/webgpu/api/validation/encoding/encoder_state.spec.ts

```diff
@@ -110,15 +121,27 @@ g.test('call_after_successful_finish')
       usage: GPUBufferUsage.COPY_DST,
     });
 
+    const callFn = () => {
+      switch (callCmd) {
+        case 'beginComputePass':
+          encoder.beginComputePass();
+          break;
+        case 'beginRenderPass':
+          t.beginRenderPass(encoder);
+          break;
+        case 'copyBufferToBuffer':
+          encoder.copyBufferToBuffer(srcBuffer, 0, dstBuffer, 0, 0);
+          break;
+        default:
+          unreachable();
+      }
+    };
+
     if (IsEncoderFinished) {
       encoder.finish();
-      t.expectValidationError(() => {
-        encoder.copyBufferToBuffer(srcBuffer, 0, dstBuffer, 0, 0);
-      }, IsEncoderFinished);
+      t.expectValidationError(callFn);
     } else {
-      t.expectValidationError(() => {
-        encoder.copyBufferToBuffer(srcBuffer, 0, dstBuffer, 0, 0);
-      }, IsEncoderFinished);
+      callFn();
       encoder.finish();
```

Comment:
The finish call was failing because the pass is open. Pushed a commit which closes it before finishing.